### PR TITLE
UF-5506: Created service from template

### PR DESCRIPTION
**Pull request by MickeWallner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,150 @@
...
+<style>
+	.settings { border-collapse: collapse; }
+	.settings th, .settings td { padding: 5px; border: solid 1px #777; vertical-align: top }
+	.settings th { background-color: lightblue; text-transform: uppercase; font-weight: normal; text-align: left}
+	.code {font-family:Courier New,Courier,monospace; background-color:#dddddd; font-size: 14px }
+</style>
+
+<h2>pw-parking-permit</h2>
+
+<p>This is a template repository that can be used when developing processes in Camunda together with SpringBoot.</p>
```
Comment (josc04) [inline README.md:10]: Change content of readme to better describe what this process-worker-service does? Some stuff can be kept (such as description on how to start and update a process instance), but most of it describes the template and not this service.

File: pom.xml [hunk 2]
```diff
@@ -1,37 +1,125 @@
...
-	<name>template</name>
+	<groupId>se.sundsvall.parkingpermit</groupId>
+	<artifactId>pw-parking-permit</artifactId>
+	<version>0.1</version>
+	<name>pw-parking-permit</name>
+
 	<properties>
 		<!-- Pipeline properties -->
-		<docker.image.name>ms-template</docker.image.name>
+		<docker.image.name>ms-parking-permit</docker.image.name>
```
Comment (josc04) [inline pom.xml:16]: pw-parking-permit?

File: src/integration-test/java/apptest/CreateProcessIT.java [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+	@Autowired
+	private CamundaClient camundaClient;
+
+	@BeforeEach
+	void setup() {
+		setDefaultPollInterval(500, MILLISECONDS);
+		setDefaultPollDelay(Duration.ZERO);
+		setDefaultTimeout(Duration.ofSeconds(30));
+
+		await()
```
Comment (josc04) [inline src/integration-test/java/apptest/CreateProcessIT.java:40]: Check all `await()` and add `.atMost(XX, timeunit)` as the test will continue forever if no upper limit is set. In some places the upper limit has been set to 30 seconds.

File: src/main/resources/application.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  title: ${spring.application.name}
+  version: '@project.version@'
+
+#----------------------------------------
+# Camunda
+#----------------------------------------
+camunda:
+  bpm:
+    client:
+      base-url: http://localhost:8080/engine-rest
```
Comment (josc04) [inline src/main/resources/application.yaml:20]: This should not be specified here, it belongs in the .env-file (or in the specific environment yaml file)

File: src/main/resources/application.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+#----------------------------------------
+# Camunda integration settings
+#----------------------------------------
+# Rest client
+integration:
+  camunda:
+    connectTimeout: 5
+    readTimeout: 20
+    url: http://localhost:8080/engine-rest
```
Comment (josc04) [inline src/main/resources/application.yaml:42]: Same here

File: README.md (hunk 1 above)
Comment (MickeWallner) [reply README.md:10]: Will do this. I thought it was not in scope for this task but it should be done anyway.

File: src/main/resources/application.yaml (hunk 4 above)
Comment (MickeWallner) [reply src/main/resources/application.yaml:20]: From camunda-template. We should change there aswell.

File: src/main/resources/application.yaml (hunk 5 above)
Comment (MickeWallner) [reply src/main/resources/application.yaml:42]: From camunda-template. We should change there aswell.

**Review by josc04 — APPROVED** [review]

**Pull request merged**